app/src/pages/admin/projectStatusPage/statusPageContent/widgets/investigated/investigated.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { defineMessages, injectIntl } from 'react-intl';
+import { InvestigatedTrendChart } from 'components/widgets/charts/investigatedTrendChart';
+import { NoItemMessage } from 'components/main/noItemMessage';
+import { mapMinListLength } from 'components/widgets/charts/investigatedTrendChart/statusPageModeConfig';
+import styles from './investigated.scss';
+
+const cx = classNames.bind(styles);
+
+const messages = defineMessages({
+  noDataMessage: {
+    id: 'LastLaunch.noDataMessage',
+    defaultMessage: 'No launches were performed',
+  },
+});
+
+@injectIntl
+export class Investigated extends Component {
+  static propTypes = {
+    data: PropTypes.object.isRequired,
+    intl: PropTypes.object.isRequired,
+    interval: PropTypes.string,
+  };
+
+  static defaultProps = {
+    interval: '3M',
+  };
+
+  state = {
+    containerEl: null,
+  };
+
+  componentDidMount() {
+    // eslint-disable-next-line react/no-did-mount-set-state
+    this.setState({ containerEl: this.myRef.current });
```

Comment:
Is this field will change its value across component life cycle?
I think it will be better to use class field property instead.

Reply:
This field changes only ones after the first render. There is only 1 alternative way to implement this: to create additional wrapper component around this component to pass only 1 property down.

Reply:
Where does it change? Usually refs are not placed in state.

Reply:
Where does it change? Usually refs are not placed in state.

Reply:
I don't have a ref at first render when I'm trying to pass ref to the inner component. Ref is being created after render is finished. Do you suggest to add an additional wrapper to pass updated ref as property down?

Reply:
I think some flag would be better